medusa/providers/generic_provider.py

```diff
@@ -645,9 +645,9 @@ def _get_season_search_strings(self, episode):
             episode_string = show_name + self.search_separator
 
             if episode.series.air_by_date or episode.series.sports:
-                episode_string += str(episode.airdate).split('-')[0]
+                search_string['Season'].append(episode_string + str(episode.airdate).split('-')[0])
```

Comment:
This I needed fix. Got broken few commits back. With the season search templates.